### DISCUSSION TITLE
GitLens Compare ... family of commands no longer grabbing focus

### DISCUSSION THREAD

**Discussion by Dzejkop** [gitkraken/vscode-gitlens#2332] [Feedback]
It used to be the case that when running, for example, `GitLens: Compare Working tree with...` I could immediately start typing out the name of the branch that I want to compare against.

Recently that stopped being the case and I'm wondering if it's something that changed in a recent update of this extension or maybe an update of VSCode.

Did anybody else notice this? Do you know if it's possible to force this behaviour with some configuration?

**Comment by eamodio** [comment]
@TylerLeonhardt Any idea on this one? I can reproduce it, but only on this quick pick where I show the quick pick early and continue to load the items after showing it.

https://github.com/gitkraken/vscode-gitlens/blob/16cdd1091beb1d5c11b127b2b8cffbfa4baea770/src/quickpicks/referencePicker.ts#L39-L161

**Comment by TylerLeonhardt** [comment]
I see you're using `enabled`... I wonder if this is https://github.com/microsoft/vscode/issues/167478

**Reply by eamodio** [reply]
Yeah, that was the cause. I ended up removing the changes to the enabled state in these quick picks to avoid it. Thankfully they really didn't *have* to be disabled while we loaded things.

**Reply by TylerLeonhardt** [reply]
I'll try to get that bug sorted this month
